features/cli/oc_expose.feature

```diff
@@ -50,12 +50,13 @@ Feature: oc_expose.feature
       | resource | secret/ssl-key   |
     Then the step should succeed
     Given evaluation of `File.read("tls.crt")` is stored in the :crt clipboard
-    And evaluation of `secret('ssl-key').created` is stored in the :birth clipboard
+    And evaluation of `v = secret('ssl-key').created; (v.is_a? Time) ? v : Time.parse(v)` is stored in the :birth clipboard
```

Comment:
creationTimestamp changes to be String instead of Time:
Before it is Time: `creationTimestamp: 2019-03-20T10:01:43Z`. Recently it changes to String: `creationTimestamp: "2019-04-11T02:58:00Z"`. Above fix is compatible no matter it is Time or String.

Reply:
Without above fix, there is failure because `cb.birth` changes to String now:
```
    And evaluation of `(cb.birth + (cb.t2 - cb.t1) + 3600 + 60).utc.strftime "%Y-%m-%dT%H:%M:%SZ"` is stored in the :newexpiry clipboard # features/step_definitions/common.rb:128
      no implicit conversion of Float into String (TypeError)
      /home/jenkins/workspace/Runner-v3/features/step_definitions/common.rb:130:in `eval'
      (eval):1:in `block in <top (required)>'
      /home/jenkins/workspace/Runner-v3/features/step_definitions/common.rb:130:in `eval'
      /home/jenkins/workspace/Runner-v3/features/step_definitions/common.rb:130:in `/^evaluation of `(.+?)` is stored in the(?: :(\S+))? clipboard$/'
      features/cli/oc_expose.feature:55:in `And evaluation of `(cb.birth + (cb.t2 - cb.t1) + 3600 + 60).utc.strftime "%Y-%m-%dT%H:%M:%SZ"` is stored in the :newexpiry clipboard'
```

Reply:
if this is v4 only, you can hardcode to parse because this branch is for 4.x only.